### PR TITLE
[PT] added portuguese translation to mix-tasks section

### PR DESCRIPTION
**Pull request by capaci**
Signed-off-by: capaci rafaelcp123@gmail.com

Related to #355.

### CODE REVIEW

File: pt/lessons/basics/mix-tasks.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+O Mix é bastante amigável por padrão. Ele sabe que todos podem ter um erro de ortografia, então ele usa uma técnica chamada "fuzzy string matching" (strings difusas, em uma tradução literal) para fazer recomendações:
+
+```shell
+$ mix hell
+** (Mix) The task "hell" could not be found. Did you mean "hello"?
+```
+
+Você notou que nós introduzimos um novo atributo, `@shortdoc`, no módulo?
+Isto facilita quando a aplicação está pronta. Por exemplo, quando um usuário executa o comando `mix help`.
```
Comment (anderkonzen) [inline pt/lessons/basics/mix-tasks.md]: Maybe ending with "executa o comando mix help no terminal." is better?

Comment (capaci) [inline pt/lessons/basics/mix-tasks.md]: you'are right, i'll fix that

File: pt/lessons/basics/mix-tasks.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+## Tarefas Mix em Ação
+
+Vamos verificar nossa tarefa Mix. Enquanto estivermos no diretório, ela deve funcionar. Na linha de comando, digite `mix hello` e então, deveremos ver o seguinte:
+
+```shell
+$ mix hello
+Hello, World!
+```
+
+O Mix é bastante amigável por padrão. Ele sabe que todos podem ter um erro de ortografia, então ele usa uma técnica chamada "fuzzy string matching" (strings difusas, em uma tradução literal) para fazer recomendações:
```
Comment (capaci) [inline pt/lessons/basics/mix-tasks.md]: @anderkonzen, about the notes in parenthesis, I had this doubt, I think the translation from "fuzzy string matching" to "strings difusas" does not make sense and it'd be better let the original.
Maybe removing the note in parenthesis would be better. What do you think?

**Comment by capaci** [comment]
@anderkonzen, thanks for the review you've made, this helps a lot. I fixed everything, except the note in parenthesis question :smile:

**Comment by ybur-yug** [comment]
If someone finalizes the OK on proofreading I'll merge this :)

Just wanna be absolutely sure because I obviously don't know the language so the more eyes the better :P

File: pt/lessons/basics/mix-tasks.md (hunk 2 above)
Comment (anderkonzen) [inline pt/lessons/basics/mix-tasks.md]: Yeah, I think so... Personally I've never heard of strings difusas 😆 in Portuguese.

Comment (capaci) [inline pt/lessons/basics/mix-tasks.md]: HAHA I fixed removing the note. Thanks!

**Comment by capaci** [comment]
@ybur-yug finished, you can merge that ;)

Also thanks, @anderkonzen, for your reviews.

**Comment by doomspork** [comment]
@pragmaticivan did you want to take a peek at this?

File: pt/lessons/basics/mix-tasks.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+* creating my_phoenix_app/config/prod.exs
+* creating my_phoenix_app/config/prod.secret.exs
+* creating my_phoenix_app/config/test.exs
+* creating my_phoenix_app/lib/my_phoenix_app.ex
+* creating my_phoenix_app/lib/my_phoenix_app/endpoint.ex
+* creating my_phoenix_app/test/views/error_view_test.exs
+...
+```
+
+Como podemos ver no comando shell acima, o Framework Phoenix tem uma tarefa Mix customizada para criar um novo projeto. E se quiséssemos criar algo parecido para o nosso projeto? Bem, a boa notícia é que nós podemos, e o Elixir permite fazermos isso de modo fácil.
```
Comment (pragmaticivan) [inline pt/lessons/basics/mix-tasks.md]: I think would be better `Elixir nos permite fazer isso de um modo fácil`

File: pt/lessons/basics/mix-tasks.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+Your Mix project was created successfully.
+You can use "mix" to compile it, test it, and more:
+
+cd hello
+mix test
+
+Run "mix help" for more commands.
+```
+
+Agora, no arquivo **lib/hello.ex** que o Mix criaou para nós, vamos criar uma função simples que irá imprimir "Hello, World!"
```
Comment (pragmaticivan) [inline pt/lessons/basics/mix-tasks.md]: `criaou` => `criou`

**Comment by doomspork** [comment]
Thank you @pragmaticivan!

**Comment by capaci** [comment]
Done! Sorry for the delay and thanks @anderkonzen and @pragmaticivan for the reviews ;)

**Comment by doomspork** [comment]
Is this good to go now @pragmaticivan?

**Pull request merged**